README.md

```diff
@@ -1,168 +1,187 @@
-# Ajax and Callbacks
+# jQuery Ajax and Callbacks
+
 ## Objectives
+
 - Explain what an API is and why it's used
 - Explain what Ajax is and why it's used
 - Make a get request using Ajax to append text to a page
 - Explain what a callback is and why Ajax runs asynchronously
 
-## Intro
-
-So far, you've been using a gem to access a web API.  Gems allow us to easily access/modify the data that's important to us. But what happens if the API we want to use doesn't have an associated gem?  We will need to handle everything ourselves. If that's the case, we'd better learn more about what an API really is!
-
-## API Basics
+## Introduction
 
-We interact with web APIs through a set of urls. Each url defines a resource that we access.  The benefit of using a gem is that we don't have to build these urls ourselves.  The gems wrap the necessary code into neat little packages.  When we ask a gem for data, a gem will make a request to the API, handle the response and parse the data into a format we can use.
+Using `XMLHttpReqest` directly is a handy way to access remote data, or web APIs, but it requires us to do a lot of lower-level wiring to get everything working. Now we're going to take a look at the Ajax capabilities of jQuery, which abstracts the XHR code into a nice, higher-level package.
```

Comment:
Maybe we can make this a bit more abstract by changing the first sentence to some thing like,

"In JavaScript, we can use `XMLHttpRequest` directly to access remote data, or web APIs, but..."